src/components/file-upload-delete/file-upload-delete.stories.mdx

```diff
@@ -0,0 +1,55 @@
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { FileUploadDelete } from "..";
+
+<Meta
+  title="Forms/FileUploadDelete"
+  component={FileUploadDelete}
+  argTypes={{
+    buttonText: {
+      control: "text",
+      description: "Text to display inside upload button",
+    },
+    helperText: {
+      control: "text",
+      description: "Text to display under upload button",
+    },
```

Comment:
I wonder if it's worth adding the onClick args here too? We can do it like: 

`onClick: {
      description: 'Callback fired when the button is clicked.',
      table: {
        type: {
          summary: 'func',
        },
      }
    },
`

Then at least the user knows the additional props you can pass in?

Reply:
Sure!